sdks/python/apache_beam/runners/portability/stager.py

```diff
@@ -897,10 +901,10 @@ def _create_beam_sdk(sdk_remote_location, temp_dir):
   def _download_pypi_sdk_package(
       temp_dir,
       fetch_binary=False,
-      language_version_tag='27',
+      language_version_tag='39',
```

Comment:
doesn't matter here, but in general changing default values may be a risky change in case there are some callers that rely on older default.

Reply:
Ack.